### PR TITLE
Add Hijri Events

### PR DESCRIPTION
**Pull request by IamRezaMousavi**
Add Hijri Events

ref: https://www.time.ir/fa/eventyear-%D8%AA%D9%82%D9%88%DB%8C%D9%85-%D8%B3%D8%A7%D9%84%DB%8C%D8%A7%D9%86%D9%87

### CODE REVIEW

**Comment by omid** [comment]
We don't have "2 Eyde Fitr"s

File: PersianCalendar@oxygenws.com/events/iranLunar.js [hunk 1]
```diff
@@ -9,20 +13,47 @@ var iranLunar = class {
...
+        this.events[9][15] = ['ولادت امام حسن مجتبی (ع)', false];
+        this.events[9][18] = ['شب قدر', false];
+        this.events[9][19] = ['ضربت خوردن امام علی (ع)', false];
+        this.events[9][20] = ['شب قدر', false];
+        this.events[9][21] = ['شهادت حضرت علی (ع)', true];
+        this.events[9][22] = ['شب قدر', false];
         this.events[10][1] = ['عید فطر', true];
-        this.events[10][2] = ['فردای عید فطر!', true];
-        this.events[10][25] = ['شهادت امام جعفر صادق', true];
+        this.events[10][2] = ['عید فطر', true];
```
Comment (omid) [inline PersianCalendar@oxygenws.com/events/iranLunar.js:47]: We don't have "2 Eyde Fitr"s

**Comment by omid** [comment]
What is the license of using that page as source? If it's not there available on their page, please ask them for permission.

File: PersianCalendar@oxygenws.com/events/iranLunar.js (hunk 1 above)
Comment (IamRezaMousavi) [reply PersianCalendar@oxygenws.com/events/iranLunar.js:47]: We have 3 or 4 days for "Eyde Nowruz"
Also have 2 days for "Eyde Fitr"

But I will solve it :)

Comment (omid) [reply PersianCalendar@oxygenws.com/events/iranLunar.js:47]: Also in your source the names are different for Fitr.
And similar for nowruz

**Comment by IamRezaMousavi** [comment]
I found a better source
The problem is solved

**Review by omid — APPROVED** [review]

**Comment by omid** [comment]
LGTM

**Pull request merged**